client/layout/masterbar/masterbar-menu.tsx

```diff
@@ -0,0 +1,46 @@
+import page from 'page';
+import { useEffect, useState } from 'react';
+import type { ReactNode, FC } from 'react';
+
+export const MasterBarMobileMenu: FC< {
+	children: ReactNode;
+	open: boolean;
+	onClose: () => void;
+} > = ( { children, open, onClose } ) => {
+	const currentURL = page.current;
```

Comment:
Do we really need to pull the entire `page` module just to fetch the current path? Could we use something built-in, like `window.location.pathname` for example?